### PR TITLE
OCLOMRS-361: Disable create button when creating a concept

### PR DESCRIPTION
**Pull request by dessHub**
… new concept

# JIRA TICKET NAME:
[OCLOMRS-361: Disable create button through the process of creating a new concept](https://issues.openmrs.org/browse/OCLOMRS-361)

# Summary:
- Reduced the setTime out as the store was being updated in a shorter time than the set timeout.

### CODE REVIEW

File: src/components/dictionaryConcepts/containers/CreateConcept.jsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ export class CreateConcept extends Component {
     }));
   }
 
-  handleChange(event) {
+   handleChange(event) {
```
Comment (collinewait) [inline src/components/dictionaryConcepts/containers/CreateConcept.jsx]: I think you can revert this change.

Comment (dessHub) [reply src/components/dictionaryConcepts/containers/CreateConcept.jsx]: done

File: src/components/dictionaryConcepts/containers/CreateConcept.jsx [hunk 2]
```diff
@@ -93,9 +93,9 @@ export class CreateConcept extends Component {
     const isAddedConcept = addedConcept.length;
     if (isNewConcept && isAddedConcept) {
       setTimeout(() => {
-        notify.show('concept successfully created', 'success', 3000);
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/containers/CreateConcept.jsx]: How does this change disable the create button?

Comment (dessHub) [reply src/components/dictionaryConcepts/containers/CreateConcept.jsx]: It doesn't. The reason I did change that was to redirect to the concepts page first, then alert the success message.

Comment (dkayiwa) [reply src/components/dictionaryConcepts/containers/CreateConcept.jsx]: In other words, is this change related to the ticket?

Comment (dessHub) [reply src/components/dictionaryConcepts/containers/CreateConcept.jsx]: Yes, it is. I realized that props were being updated faster than the given timeout hence the button is being re-enabled before the page redirects.
